src/platform/datapath_winkernel.c

```diff
@@ -796,6 +796,10 @@ CxPlatDataPathQuerySockoptSupport(
 
     } while (FALSE);
 
+#if defined(QUIC_API_ENABLE_PREVIEW_FEATURES)
+    Datapath->Features |= CXPLAT_DATAPATH_FEATURE_TCP;
```

Comment:
Why setting the TCP flag?

Reply:
oh, which datapath support TCP (QTIP)? only XDP?

Reply:
only winxdp supports QTIP.